src/main/scala/com/databricks/spark/sql/perf/table.scala

```diff
@@ -47,7 +35,7 @@ abstract class TableForTest(
 
   val name = table.name
 
-  val outputDir = s"$baseDir/parquet/${name}"
+  val outputDir = s"$baseDir/${name}"
```

Comment:
Good point! @yhuai @marmbrus ? 
The alternative being that we could make sure the kms-encrypted dataset has the baseDir/parquet/ structure. So that this change is not needed.